lib/src/sdk_api/view/video_view.dart

```diff
@@ -105,18 +101,20 @@ class VideoView extends StatefulWidget {
   const VideoView.withMediaStream(
       {required this.participant,
       required this.mediaStream,
-      this.scaleType,
```

Comment:
Does this have to be an optional? Can a default value not be provided?

Would following not work?

```dart
const VideoView.withMediaStream({
       required this.participant,
       required this.mediaStream,
       ScaleType scaleType = ScaleType.fill,
       Key? key})
```

Reply:
Default value is set when value is not provided. In above example scaleType will be always ScaleType.fill.

Reply:
ScaleType could be required, but then you have to provide scaleType and default value is not needed.